common-design/css/styles.css

```diff
@@ -427,7 +475,6 @@ ul.cd-nav__grandchild-nav {
       display: -webkit-flex;
       display: flex;
       height: 60px;
-      line-height: 60px;
```

Comment:
this line didn't add anything, and it messed up the squished nav links when they were forced to wrap as flex-items.